src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+    },
 
+    divide(number) {
+      calculator.result /= number;
```

Comment:
don't interact with the object's result, just return calculation and set it in operate function

Reply:
could you help me, how i can do it? I little understand all how work it